cmake/Hip.cmake

```diff
@@ -112,9 +112,16 @@ find_package(HIP 1.0)
 IF(HIP_FOUND)
   set(HAVE_HIP TRUE)
 
+  if(HIP_COMPILER STREQUAL clang)
```

Comment:
Will it be a good idea to define HIP_COMPILER in gloo also, in case gloo is independently compiled?

Reply:
HIP_COMPILER is set by `find_package(HIP 1.0)`.  The FindHIP.cmake does this.